ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/BlockSimulator.java

```diff
@@ -68,6 +73,18 @@
  * header, transaction receipts, and other relevant information.
  */
 public class BlockSimulator {
+
+  private static final ImmutableTransactionValidationParams STRICT_VALIDATION_PARAMS =
+      ImmutableTransactionValidationParams.builder()
+          .from(TransactionValidationParams.transactionSimulator())
+          .build();
```

Comment:
It seems you are only aliasing an existing object, so a little optimization
```suggestion
  private static final TransactionValidationParams STRICT_VALIDATION_PARAMS =
     TransactionValidationParams.transactionSimulator());
```

Reply:
done